modules/ui/CaptionUI.py

```diff
@@ -163,7 +172,7 @@ def content_column(self, master):
         # prompt
         self.prompt_var = ctk.StringVar()
         self.prompt_component = ctk.CTkEntry(right_frame, textvariable=self.prompt_var)
-        self.prompt_component.grid(row=2, column=0, pady=5, sticky="new")
+        self.prompt_component.grid(row=2, column=0, columnspan=3, pady=5, sticky="new")
         self.prompt_component.bind("<Down>", self.next_image)
```

Comment:
Here, all the shortcuts are bound to the prompt input field. Until now, this was the only component that could receive input, so these shortcuts always worked. But now there is a second input field. Can you bind the shortcuts to both input fields?

Reply:
Done